website/docs/basics/blocks-component-in-block.md

```diff
@@ -342,3 +342,60 @@ echo wp_kses_post(
 	)
 );
 ```
+
+### I want to limit which options are shown for components inside a block
+
+Let's say you have a block that has a `Heading` component inside it.
+
+The `Heading` has 10 text sizes and 5 colors, but for that block only 2 colors and 3 text sizes are allowed.
+
+With the `filterComponentOptions` helper you can limit which options are shown in the editor.
+
+Implementing it is easy:
+1) In your **block** manifest add a key inside `options` that contains all the options you want visible. Options that are not added will use all the options defined in the component's manifest.
+
+   For example:
+   ```json
+   "options" : {
+	   "heading": {
+		   "colors": [
+			   "primary",
+			   "black"
+		   ],
+		   "sizes": [
+			   "xxl",
+			   "xl"
+		   ]
+	   }
+   }
+   ```
+	❗️ **Note**
```

Comment:
```suggestion
> In case your option is an object with a `label` and a `value` key (eg. when you have something in a select menu), you only need to provide the `value`.
```